core/domain/collection_domain.py

```diff
@@ -1441,42 +1068,25 @@ def validate(self, strict=True):
                     'collection.')
 
             # Ensure the collection may be started.
-            if not self.init_exploration_ids:
+            if not self.init_exploration_id:
                 raise utils.ValidationError(
-                    'Expected to have at least 1 exploration with no '
-                    'prerequisite skill ids.')
-
-            # Ensure the collection can be completed. This is done in two
-            # steps: first, no exploration may grant a skill that it
-            # simultaneously lists as a prerequisite. Second, every exploration
-            # in the collection must be reachable when starting from the
-            # explorations with no prerequisite skill ids and playing through
-            # all subsequent explorations provided by get_next_exploration_ids.
-            completed_exp_ids = set(self.init_exploration_ids)
-            next_exp_ids = self.get_next_exploration_ids(
-                list(completed_exp_ids))
-            while next_exp_ids:
-                completed_exp_ids.update(set(next_exp_ids))
-                next_exp_ids = self.get_next_exploration_ids(
-                    list(completed_exp_ids))
+                    'Expected to have at least 1 exploration.')
+
+            # Ensure the collection can be completed. In the current
+            # implementation, without skills, this would always be true.
```

Comment:
Yeah, you can just remove all this I think.

Reply:
Done.